src/main/java/org/broadinstitute/hellbender/utils/baq/BAQ.java

```diff
@@ -662,7 +662,7 @@ public byte capBaseByBAQ( byte oq, byte bq, int state, int expectedPos ) {
                     }
                 } else if ( readHasBAQTag ) {
                     // remove the BAQ tag if it's there because we cannot trust it
-                    read.setAttribute(BAQ_TAG, null);
+                    read.clearAttribute(BAQ_TAG);
```

Comment:
i like this!  `read.setAttribute(BAQ_TAG, null)` -> ` read.clearAttribute(BAQ_TAG)`